fapolicy_analyzer/glade/main_window.glade

```diff
@@ -249,6 +282,63 @@
                 <property name="position">2</property>
               </packing>
             </child>
+            <child>
+              <object class="GtkStatusbar">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="margin-left">10</property>
+                <property name="margin-right">10</property>
+                <property name="margin-start">10</property>
+                <property name="margin-end">10</property>
+                <property name="margin-top">6</property>
+                <property name="margin-bottom">6</property>
+                <property name="spacing">2</property>
+                <child>
+                  <object class="GtkProgressBar" id="fapaStatusProgressBar">
```

Comment:
What is this progress bar doing?  I don't see any references to it.

Reply:
It was an aspirational place-holder. I thought we could use it at some point as opposed to having the single fapd status icon. Removed it.